operate/services/protocol.py

```diff
@@ -385,7 +385,10 @@ def check_if_unstaking_possible(
                 self.ledger_api, staking_contract
             ).get("data"),
         )
-        staked_duration = time.time() - ts_start
+        
+        current_block = self.ledger_api.api.eth.get_block('latest')
+        current_timestamp = current_block.timestamp
+        staked_duration = current_timestamp - ts_start
```

Comment:
It's a good change but can you make this change directly, on `fix/meme-staging`?

Reply:
resolved https://github.com/valory-xyz/olas-operate-app/pull/722